### PR TITLE
Bump to 2.9.0

### PR DESCRIPTION
**Pull request by ericdill**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
   entry_points:
     - depfinder = depfinder.cli:cli
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv
   number: 0
 
 requirements:
   host:
-    - python >=3.6,<3.10
+    - python >=3.6
```
Comment (ocefpaf) [inline recipe/meta.yaml:19]: Sorry @ericdill, missed this one. I'm so used to just fix the bot's PR with grayskull that I did not pay attention to a human PR. The only issue here is that we need to remove the `>=3.6` but that is already done in #30.

**Comment by ocefpaf** [comment]
#30 implemented the changes here. Closing this.

**Comment by ericdill** [comment]
Thanks for fixing up my blunder here 🤜🤛

**Comment by ocefpaf** [comment]
> Thanks for fixing up my blunder here fist_rightfist_left

BTW, I don't know how to fix these things up anymore, I'm quite outdated. I just regenerate with grayskull and hope for the best. So far it never disappointed me.

**Pull request closed without merging**